src/psyclone/psyir/symbols/datatypes.py

```diff
@@ -193,13 +193,13 @@ def type_text(self):
         # pylint:disable=import-outside-toplevel
         from fparser.common.readfortran import FortranStringReader
         from fparser.common.sourceinfo import FortranFormat
-        from fparser.two import Fortran2008
+        from fparser.two import Fortran2003
```

Comment:
I'm not sure whether or not we should make classes that are common to both F08 and F03 available via either module. However, that's not something to worry about here.